File: ServerSpace/src/main/java/Business/Moneda.java
package Business;

import DataLayer.MonedaDB;
import EntityLayer.MonedaEntity;
import Framework.Inj;
import java.util.ArrayList;

public class Moneda {
    
    
    public ArrayList<MonedaEntity> ObtenerMonedaItems() {
        try {
            MonedaDB BD = new MonedaDB();
            return BD.ObtenerMonedaItems();
        } catch (Exception ex) {
            System.out.println("ERROR " + ex);
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }

    public ArrayList<MonedaEntity> ObtenerMonedaItem(int Id) {
        try {
            MonedaDB BD = new MonedaDB();
            return BD.ObtenerMonedaItem(Id);
        } catch (Exception ex) {
            System.out.println("ERROR " + ex);
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }

}

File: SharedApi/Space/Space.BusinessLayer/Moneda.cs
using Space.DataLayer;
using Space.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Space.BusinessLayer
{
    public class Moneda
    {
        public static List<MonedaEntity> ObtenerMonedaItems()
        {
            MonedaDB DB = new MonedaDB();
            return DB.ObtenerMonedaItems();
        }

        public static List<MonedaEntity> ObtenerMonedaItem(Int32 MonedaId)
        {
            MonedaDB DB = new MonedaDB();
            return DB.ObtenerMonedaItem(MonedaId);
        }

    }
}
